### ISSUE TITLE
Cannot view rules details on Server Rules menu

### ISSUE THREAD

**Issue by TK-A369** [space-wizards/space-station-14#34131]
## Description
After not playing for few months, I joined WizDen Salamander server, and it shown the "Server Rules" window, as expected. Clicking on individual rules used to show detailed description of them, along with some examples. However, now it doesn't happen. Clicking on them does nothing, even though they are highlighted when you hover mouse over them, like hyperlinks, suggesting that they're clickable.

**Reproduction**
1. Join server you haven't been on for a long time (I guess you can somehow make server forget your last visit, for testing purposes, right?)
2. As a law-abiding ~citizen~ player, read the rules
3. Click on any rule - it doesn't show its details

**Screenshots**
![Screenshot](https://github.com/user-attachments/assets/05f20b22-3142-4cb6-9c42-ae6e05a176cc)

**Additional context**
I didn't test it more thoroughly (yet).

**Comment by TK-A369** [comment]
Oh, after clicking "I have read and agree to follow the rules", it showed the guidebook window open on the rule I clicked.
But it's still not how it's supposed to work. I believe it used to either open it in popup window (but visible immediately), or inside this frame, with "go back" button.

**Comment by Alpaccalypse** [comment]
Yeah, I can replicate this. It is particularly frustrating, as it only unlocks if you confirm you have read the rules, which you can't do until you confirm you have.

**Comment by lzk228** [comment]
i also can confirm that it happens sometimes

**Comment by ArtisticRoomba** [comment]
Needs to be reproduced as this should have been fixed recently
